### DISCUSSION TITLE
Cannot create draft items in Projects beta on mobile

### DISCUSSION THREAD

**Discussion by borekb** [community/community#20065] [Projects and Issues]
I've tried using Projects on my mobile phone – Android with the Gboard keyboard – and pressing the main action button (which I'm not even sure is "Enter") does nothing. As a result, I was not able to add new items / drafts:

<img width="350" src="https://user-images.githubusercontent.com/101152/177416117-6e87e7f9-ec30-4f9d-8dbc-d3bdbe0a19ac.png">

I think this is a small bug.

**Comment by MrCatEsq** [comment]
Any updates or comments on this? Same problem.

**Comment by MrCatEsq** [comment]
I informed gboard-dev-support@google.com about this issue (you also can do this for some publicity effect), but can't hope on a quick response. 

Anyone tried to use alternative keyboards?

**Comment by MrCatEsq** [comment]
Found a way: after card name is entered press the space key and the right arrow key in bottom right corner in sequence (not space+enter, but terminal space then enter). Even though it looks weird, this solution works for me.
Source: https://github.com/community/community/discussions/41986#discussioncomment-4471118

**Reply by rachidlamouri** [reply]
Thank you!!!. I'm still having this problem though, so I'd love to see an official fix 🎉
